grub/install_script_grub.sh

```diff
@@ -1,23 +1,225 @@
 #!/bin/bash
-NC='\033[0m' # No Color
+
+##############################################################################
+#      Virtuaverse GRUB Theme Installation Script
+#
+#      Description:
+#      This script installs the Virtuaverse GRUB theme.
+#
+#      Authors:
+#      Federico Slongo
+#      Pewrie Bontal
+#
+#      Copyright 2022-2024 the original author or authors
+##############################################################################
+
 CYAN='\033[0;36m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+ORANGE='\033[0;33m'
+NC='\033[0m'
+
+#################################################
+#@# Check correct grub directory to install
+check_grub_ver() {
+  if [ -x "$(command -v grub2-mkconfig)" ] && [ -d "/boot/grub2" ]; then
+    if [ -x "$(command -v grub-mkconfig)" ] && [ -d "/boot/grub" ]; then
+      printf "${RED}[ERR]\tBoth Grub2 and Grub detected. IDK what to do\n"
+      exit 1
+    else
+      GRUB_DIR="/boot/grub2"
+      printf "${CYAN}[MSG]\tGrub2 detected. Using directory: $GRUB_DIR\n"
+    fi
+  elif [ -x "$(command -v grub-mkconfig)" ] && [ -d "/boot/grub" ]; then
+    GRUB_DIR="/boot/grub"
+    printf "${CYAN}[MSG]\tGrub detected. Using directory: $GRUB_DIR\n${NC}"
+  else
+    printf "${RED}[ERR]\tNo grub directory found! wtf?\n${NC}"
+    printf "${RED}[ERR]\tExiting...\n${NC}"
+    exit 1
+  fi
+
+  printf "[MSG]\tUsing grub directory: $GRUB_DIR\n"
+}
+
+check_grub_ver() {
```

Comment:
Seems to be legacy... To be removed? (or am I missing something?)

Reply:
my bad, forgot to remove sec one